plugins/woocommerce-blocks/tests/e2e/tests/product-collection/product-collection.block_theme.spec.ts

```diff
@@ -1590,6 +1590,62 @@ test.describe( 'Product Collection', () => {
 			await expect( products ).toHaveText( expectedProducts );
 		} );
 	} );
+
+	test.describe( 'Extensibility - JS events', () => {
+		test( 'emits event about block rendered', async ( {
```

Comment:
I'd use the event ID in the title - easier to find a responsible test! For example `"emits the wc-blocks_product_list_rendered event"`.